framework/packages/abstract-adapter/src/endpoints/execute.rs

```diff
@@ -154,9 +154,10 @@ impl<Error: ContractError, CustomInitMsg, CustomExecMsg, CustomQueryMsg, Receive
             }
             None => account_registry
                 .assert_manager(sender)
-                .map_err(|_| unauthorized_sender())?,
+                .map_err(|_| unauthorized_sender())
+                .ok(),
         };
-        self.target_account = Some(account_base);
+        self.target_account = account_base;
```

Comment:
This change allows calling the adapter without adding a sender. while it's a good hackathon fix the standalone module should really handle that use case. I don't think we should merge this change.